### PR TITLE
Can attach debugger with Lua code now.

### PR DESCRIPTION
**Pull request by ghost**
One has to call `dbg()` on sections of code which one wishes to debug.
Debugger used is [debugger.lua](https://github.com/slembcke/debugger.lua).

### CODE REVIEW

File: script/match.lua [hunk 1]
```diff
@@ -21,7 +23,6 @@ function Match:new (o)
     local player = Player:new(o.player_description)
     player.id = #o.entities + 1
     o.entities[player.id] = player
-
```
Comment (forestbelton) [inline script/match.lua]: Can you remove these changes? I find that adding empty newlines between code sections helps for clarity.

**Comment by LiquidLemon** [comment]
There's something worth considering - having a script that launches the debugger immediately so that it can be used with the `dofile()` function. It would make it both easier to quickly insert a breakpoint into scripts and remove all debugging utilities when no longer needed.

**Comment by ghost** [comment]
@forestbelton newlines are back

**Comment by ghost** [comment]
@LiquidLemon looking at it. It does sound like a good idea :)

**Comment by ghost** [comment]
@LiquidLemon you mean something like this?
```lua
# debug.lua
local dbg = require('script/debugger')

dbg()
dofile('script/match.lua')
```

**Pull request merged**